lib/active_fedora/file.rb

```diff
@@ -28,7 +28,7 @@ class File
     # @param parent_or_url_or_hash [ActiveFedora::Base, String, Hash, NilClass] the parent resource or the URI of this resource
     # @param path [String] the path partial relative to the resource
     # @param options [Hash]
-    def initialize(parent_or_url_or_hash = nil, path=nil, options={})
+    def initialize(parent_or_url_or_hash = nil, path=nil, options={}, &block)
```

Comment:
@jcoyne yard doc about the block would be nice.  https://gist.github.com/chetan/1827484#blocks